### PR TITLE
Add a funtion to forge Android logd entry

### PR DESCRIPTION
**Pull request by DimaWittmann**
This PR adds a function that constructs a custom log entry.
I can be used it forge log entries, for example, to forward logs for 3rd party app to lodg

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -63,6 +63,12 @@
 //! android_logd_logger::write_event_now(1, "test").unwrap();
 //! ```
 //!
+//! To forge android logd entry:
+//!
+//! ```
+//! android_logd_logger::low(buffer_id, pid, thread_id, secs, subsec_nanos, tag, priority, message).unwrap();
```
Comment (flxo) [inline src/lib.rs]: Here's a typo. Should be `log`.  Can you please also ensure that rustdoc catches the examples and tries to build them?

File: src/lib.rs [hunk 2]
```diff
@@ -490,3 +513,48 @@ impl Builder {
...
+        tag,
+        priority,
+        message,
+    };
+
+    #[cfg(target_os = "android")]
+    crate::logd::log(&record);
+
+    #[cfg(not(target_os = "android"))]
+    println!("buffer: {:?}, tag: {:?}, message: {:?}", buffer_id, tag, message);
```
Comment (flxo) [inline src/lib.rs]: Please forward the records in the not android case as well. This keeps the equals format.

**Review by flxo — CHANGES_REQUESTED** [review]
Thanks for your contribution. Left some comments.

File: src/lib.rs (hunk 1 above)
Comment (DimaWittmann) [reply src/lib.rs]: Thank you for suggestion. I checked with the rustdoc

File: src/lib.rs (hunk 2 above)
Comment (DimaWittmann) [reply src/lib.rs]: Of course, I was just not sure, I removed this conditional compilation

**Pull request merged**